image/config/nginx.conf

```diff
@@ -38,11 +44,10 @@ http {
 	ssl_prefer_server_ciphers on;
 
 	##
-	# Logging Settings
+	# Access logs
 	##
 
 	access_log /var/log/nginx/access.log;
-	error_log /var/log/nginx/error.log;
```

Comment:
is it falling back to STDOUT by default ? why this change?

Reply:
It still defines error_log /var/log/nginx/error.log;  - it just does it at the root scope now.

You can look at https://github.com/phusion/passenger/issues/2541 for why.